scripts/known-issues.sh

```diff
@@ -0,0 +1,273 @@
+#!/bin/bash
+#
+# This script will generate HTML describing which failures have
+# been observed by the buildbot when running ZFS Test Suite.  Its
+# purpose is to allow developers to quickly assess the prevalence
+# of any observed failures.
+#
+
+ZFSONLINUX_DIR="/home/buildbot/zfs-buildbot/master/*_TEST_"
+ZFSONLINUX_MTIME=30
+ZFSONLINUX_MMIN=$((ZFSONLINUX_MTIME*24*60))
+ZFSONLINUX_ISSUES=$(curl -s https://api.github.com/search/issues?q=label:%22Test%20Suite%22+type:issue+repo:zfsonlinux/zfs)
+
+NUMBER_REGEX='^[0-9]+$'
+DATE=$(date)
+STATUS_LOW_CUTOFF=5
+STATUS_MED_CUTOFF=20
+
+STATUS_LOW="st_low"
+STATUS_MED="st_med"
+STATUS_HIGH="st_high"
+STATUS_PR="st_pr"
+
+STATUS_LOW_COLOR="#ffee3a"
+STATUS_MED_COLOR="#ffa500"
+STATUS_HIGH_COLOR="#ff9999"
+STATUS_PR_COLOR="#f8f8f8"
+
+STATUS_LOW_TEXT="low"
+STATUS_MED_TEXT="medium"
+STATUS_HIGH_TEXT="high"
+STATUS_PR_TEXT=""
+
+usage() {
+cat << EOF
+USAGE:
+$0 [-h] [-d directory] [-m mtime]
+
+DESCRIPTION:
+	Dynamically generate HTML for the Known Issue Tracking page
+	using the ZFS Test Suite results from the buildbot automated
+	testing.
+
+OPTIONS:
+	-h		Show this message
+	-d directory	Directory containing the buildbot logs
+	-m mtime	Include test logs from the last N days.
+
+EXAMPLE:
+
+$0 -d ~/zfs-buildbot/master/*_TEST_ -m 30 \\
+    >~/zfs-buildbot/master/public_html/known-issues.html
+
+EOF
+}
+
+while getopts 'hd:m:' OPTION; do
+	case $OPTION in
+	h)
+		usage
+		exit 1
+		;;
+	d)
+		ZFSONLINUX_DIR=$OPTARG
+		;;
+	m)
+		ZFSONLINUX_MTIME=$OPTARG
+		ZFSONLINUX_MMIN=$((ZFSONLINUX_MTIME*24*60))
+		;;
+	esac
+done
+
+cat << EOF
+<!DOCTYPE html>
+<html>
+<head>
+<title>Known Issue Tracking</title>
+<meta name="keyword" content="zfs, linux"/>
+<meta http-equiv="Content-type" content="text/html; charset=utf-8">
+<script		  src="https://code.jquery.com/jquery-1.12.4.min.js"
+			  integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
+			  crossorigin="anonymous"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css">
+<script type="text/javascript" language="javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
+
```

Comment:
Please add here 
```
<script type="text/javascript" language="javascript" src="https://cdn.datatables.net/plug-ins/1.10.16/sorting/enum.js"></script>
```
it's a plugin to sort last column.

Reply:
Done.